src/app/CommandSender.h

```diff
@@ -461,7 +461,7 @@ class CommandSender final : public Messaging::ExchangeDelegate
         AddingCommand,       ///< In the process of adding a command.
         AddedCommand,        ///< A command has been completely encoded and is awaiting transmission.
         AwaitingTimedStatus, ///< Sent a Timed Request and waiting for response.
-        CommandSent,         ///< The command has been sent successfully.
+        AwaitingResponse,    ///< The command has been sent successfully, and we are awaiting invoke response.
```

Comment:
With suppress response == true, does the code move to AwaitingDestruction today?

Reply:
suppress response is entirely ignored on both sides right now and completely unimplemented